indico/modules/events/editing/client/js/management/editable_type/EditableList.module.scss

```
@@ -99,4 +99,16 @@ $border: 1px solid $pastel-gray;
     /* stylelint-disable-next-line value-keyword-case */
     fill: currentColor;
   }
+
+  :global(.more-indicator) {
```

Just to make it clearer that this is an indicator for the tags specifically:
```suggestion
  :global(.more-tags-indicator) {
```